src/sensors.c

```diff
@@ -155,7 +160,9 @@ void init_sensors(void)
   mpu6050_register_interrupt_cb(&imu_ISR);
 
   uint16_t acc1G;
-  mpu6050_init(true, &acc1G, &gyro_scale, _params.values[PARAM_BOARD_REVISION]);
+  float gyro_scale_to_urad;
+  mpu6050_init(true, &acc1G, &gyro_scale_to_urad, _params.values[PARAM_BOARD_REVISION]);
+  gyro_scale = 1000000.0*gyro_scale_to_urad;
```

Comment:
Right, so if we start with a value in urad, then we have `x urad * (1 rad / 1e6 urad) = x rad` (you have to _divide_ by 1e6 so that urad cancels). So if `raw * gyro_scale_to_urad = x urad`, then substituting into the previous equation we get `raw * gyro_scale_to_urad * (1 rad / 1e6 urad) = x rad`, which we can write as `raw * (gyro_scale_to_urad / 1e6) = x rad`, so we would have `gyro_scale = gyro_scale_to_urad / 1e6`. Is that right, or am I just completely losing my mind?